public/Day 141/style.css

```
@@ -473,3 +473,71 @@ h1, h2, h3, .brand {
+  content: "";
+  position: absolute;
+  width: 300%;
+  height: 300%;
+  top: -100%;
+  left: -100%;
+  background: radial-gradient(circle, rgba(255,255,255,0.2), transparent 70%);
+  animation: floatBg 18s linear infinite;
+  pointer-events: none;
+}
+
+@keyframes floatBg {
```

Stylelint (17.3.0)

[error] Expected keyframe name "floatBg" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)